.gitignore

```diff
@@ -10,3 +10,7 @@
 
 # python
 *.pyc
+
+# OSX
+.DS_Store
+
```

Comment:
please remove this change from the branch. if you would like to submit it, send as separate PR